manticore/__main__.py

```diff
@@ -95,21 +104,13 @@ def ethereum_cli(args):
                          data=symbolic_data,
                          value=symbolic_value )
 
-        tx_count += 1
         last_coverage = new_coverage
         new_coverage = m.global_coverage(contract_account)
 
-        print "[+] Coverage after %d transactions: %d%%"%(tx_count, new_coverage)
```

Comment:
we should still output the actual coverage number, let's use the `__main__` logger for the moment for this since we don't have access to it inside manticore. eventually this function will be moved inside manticore, and we can use the appropriate logger once we move it

Reply:
actually we can technically call `m.global_coverage` from within manticore to avoid logging it out here in main. hopefully it's not too slow since we're doing it twice

Reply:
Moved debug output to `transaction()` since we need access to the contract on which we're calculating coverage.